addons/point_of_sale/report/__init__.py

```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import pos_users_product
 import account_statement
-import pos_receipt
+import pos_details
 import pos_invoice
 import pos_lines
-import pos_details
-import pos_payment_report
-import pos_report
 import pos_order_report
+import pos_payment_report
+import pos_receipt
+import pos_users_product
```

Comment:
I think you forget something
```
\n
```
![forget](https://cloud.githubusercontent.com/assets/14940179/12128987/0781a7bc-b3be-11e5-9a38-ffbdc9f54119.gif)